### ISSUE TITLE
Qemu tests fail when building

### ISSUE THREAD

**Issue by SpidFightFR** [GNS3/gns3-server#2305]
Hello i encounter an error when trying to build GNS3-Server, here is the error i'm getting:

<details><summary>Error</summary>

```
=========================== short test summary info ============================
FAILED tests/compute/qemu/test_qemu_vm.py::test_reload - AttributeError: 'cal...
FAILED tests/compute/qemu/test_qemu_vm.py::test_suspend - AttributeError: 'ca...
FAILED tests/compute/qemu/test_qemu_vm.py::test_control_vm - AttributeError: ...
FAILED tests/compute/qemu/test_qemu_vm.py::test_control_vm_expect_text - Attr...
FAILED tests/handlers/api/compute/test_qemu.py::test_qemu_list_binaries - Att...
FAILED tests/handlers/api/compute/test_qemu.py::test_qemu_list_binaries_filter
====== 6 failed, 891 passed, 3 skipped, 921 warnings in 83.40s (0:01:23) =======

```
</details>

Tho i must say that it is my first time compiling the software (i always used the provided VMs, but i needed to use the bare-metal server this once. :sweat_smile: 

Thanks by advance for the help !
If any more info is required, feel free to ask !

**Comment by grossmj** [comment]
> Hello i encounter an error when trying to build GNS3-Server, here is the error i'm getting:

What commands have you used to build? What version are you trying to build?

Thanks.

**Comment by SpidFightFR** [comment]
> > Hello i encounter an error when trying to build GNS3-Server, here is the error i'm getting:
> 
> What commands have you used to build? What version are you trying to build?
> 
> Thanks.

Hello!
I try to update the gns3-server package for the voidlinux distribution. So i used their utility xbps-src.

I don't know precisely what it uses to build, all i can say is that gns3-server is built as a python3-module, which worked up until then.

I think [this](https://github.com/void-linux/void-packages/actions/runs/6670200550/job/18129590490?pr=46778) can help, it's the building logs of gns3-server (where i took the error logs i sent here).

I am sorry if that isn't really useful! 😅

**Comment by grossmj** [comment]
I can see the tests were run with Python 3.12. We have fixed this recently https://github.com/GNS3/gns3-server/commit/34ee75e1f1eabb7d24f035d3deeddf6ca5a3a9ea

Things should work properly in v2.2.44

**Comment by SpidFightFR** [comment]
> I can see the tests were run with Python 3.12. We have fixed this recently [34ee75e](https://github.com/GNS3/gns3-server/commit/34ee75e1f1eabb7d24f035d3deeddf6ca5a3a9ea)
> 
> Things should work properly in v2.2.44

Thank you very much for your help ! i'll try to modify the package for the 2.2.44 version as soon as possible then! :smile:
